apollo-router/src/plugins/connectors/tests.rs

```diff
@@ -705,11 +705,292 @@ async fn test_no_source() {
+                },
+                {
+                  "userId": 1,
+                  "id": 2,
+                  "title": "qui est esse",
+                  "body": "est rerum tempore vitae\nsequi sint nihil reprehenderit dolor beatae ea dolores neque\nfugiat blanditiis voluptate porro vel nihil molestiae ut reiciendis\nqui aperiam non debitis possimus qui neque nisi nulla"
+                }]
+            )),
+        ).mount(&mock_server).await;
+        Mock::given(method("GET")).and(path("/posts/1")).respond_with(
+            ResponseTemplate::new(200).set_body_json(serde_json::json!(
+                {
+                  "userId": 1,
+                  "id": 1,
+                  "title": "sunt aut facere repellat provident occaecati excepturi optio reprehenderit",
+                  "body": "quia et suscipit\nsuscipit recusandae consequuntur expedita et cum\nreprehenderit molestiae ut ut quas totam\nnostrum rerum est autem sunt rem eveniet architecto"
+                }
+            )),
+        ).mount(&mock_server).await;
+        Mock::given(method("GET")).and(path("/posts/2")).respond_with(
+            ResponseTemplate::new(200).set_body_json(serde_json::json!({
+                  "userId": 1,
+                  "id": 2,
+                  "title": "qui est esse",
+                  "body": "est rerum tempore vitae\nsequi sint nihil reprehenderit dolor beatae ea dolores neque\nfugiat blanditiis voluptate porro vel nihil molestiae ut reiciendis\nqui aperiam non debitis possimus qui neque nisi nulla"
+                }
+            )),
+        ).mount(&mock_server).await;
+        Mock::given(method("GET"))
+            .and(path("/users/1"))
+            .respond_with(ResponseTemplate::new(200).set_body_json(serde_json::json!({
+              "id": 1,
+              "name": "Leanne Graham",
+              "username": "Bret",
+              "email": "Sincere@april.biz",
+              "address": {
+                "street": "Kulas Light",
+                "suite": "Apt. 556",
+                "city": "Gwenborough",
+                "zipcode": "92998-3874",
+                "geo": {
+                  "lat": "-37.3159",
+                  "lng": "81.1496"
+                }
+              },
+              "phone": "1-770-736-8031 x56442",
+              "website": "hildegard.org",
+              "company": {
+                "name": "Romaguera-Crona",
+                "catchPhrase": "Multi-layered client-server neural-net",
+                "bs": "harness real-time e-markets"
+              }
+            })))
+            .mount(&mock_server)
+            .await;
+        Mock::given(method("GET")).and(path("/users/1/posts")).respond_with(
+            ResponseTemplate::new(200).set_body_json(serde_json::json!([
+                {
+                  "userId": 1,
```

Comment:
Extra attributes shouldn't hurt, but it shouldn't _need_ to include `userId`, right?

Reply:
these are straight copies from jsonplaceholder (except with shorter lists)